lib/node/http.js

```diff
@@ -125,6 +125,7 @@ class HTTP extends Server {
           height: this.chain.height,
           tip: this.chain.tip.hash.toString('hex'),
           treeRoot: this.chain.tip.treeRoot.toString('hex'),
```

Comment:
This could influence people towards using the chain in an unsafe way. It could be useful for block explorers though.
It could also be useful in conjunction with the `getnameproof` RPC